src/scribe_data/cli/convert.py

```diff
@@ -207,13 +202,10 @@ def convert_to_csv_or_tsv(
     ----------
     language : str
         The language of the file to convert.
-
```

Comment:
I'd prefer to keep these spaces so the docstrings aren't as dense :) For me makes it easier to read them 😊